app/swagger/schemas/vet360/telephone.rb

```diff
@@ -8,67 +8,137 @@ class Telephone
 
         swagger_schema :PostVet360Telephone do
           key :required, %i[phone_number area_code phone_type is_international country_code]
-          property :is_international,
-                   type: :boolean,
-                   enum: [false],
-                   example: false
-          property :country_code,
-                   type: :string,
-                   enum: ['1'],
-                   example: '1'
-          property :phone_number,
-                   type: :string,
-                   example: '5551212',
-                   minLength: 1,
-                   maxLength: 14,
-                   pattern: ::Vet360::Models::Telephone::VALID_PHONE_NUMBER_REGEX.inspect
           property :area_code,
                    type: :string,
                    example: '303',
                    minLength: 3,
                    maxLength: 3,
                    pattern: ::Vet360::Models::Telephone::VALID_AREA_CODE_REGEX.inspect
+          property :country_code,
+                   type: :string,
+                   enum: ['1'],
+                   example: '1'
+          # property :created_at,
+          #          type: :string,
+          #          format: 'date-time',
+          #          example: '2018-04-21T20:09:50Z'
+          property :effective_end_date,
```

Comment:
I think that we want to take this param out of the documentation because it's always removed or overridden by our code.